rpcq/__init__.py

```diff
@@ -1,3 +1,9 @@
 from rpcq._client import Client, ClientAuthConfig
 from rpcq._server import Server, ServerAuthConfig
+# These are imported so that the corresponding data classes are
+# registered whenever rpcq is imported. Without which one would have
+# to import the messages and core_messages modules directly before
+# using, e.g., from_json / to_json.
+from rpcq import messages
+from rpcq import core_messages
```

Comment:
If the intention isn't to expose `message` and `core_messages`, maybe `from rpcq import messages as _messages, core_messages as _core_messages`.

Reply:
In what sense are `messages` and `core_messages` exposed? If I understand correctly, if I `import rpcq` (assuming the above changes) then I don't import `messages.*` and `core_messages.*` into the `rpcq` namespace. I *do* have access to `rpcq.messages` and `rpcq.core_messages` but that is true even without the above changes.